nginx/config.go

```diff
@@ -293,6 +293,17 @@ http {
 		proxy_pass {{$builderConfig.ServiceIP}}:2222;
 	}
 }{{ end }}
+
+{{ if $routerConfig.StatusEndpoint }}
+  sevrer {
```

Comment:
Also, probably this shouldn't be its own vhost / server block. As it stands, I don' think any traffic will ever reach this endpoint. The router uses hostname-based routing... so every request either goes to an application _or_ to the default vhost. This new server block is neither of those things. Probably it would be better to add this to the existing default vhost. Once you do, the only way to access this endpoint will be by making a request to the router that doesn't match any other application-- e.g. `bogus.example.com/nginx_status`. Is that suitable?

Or I'm wondering if maybe we shouldn't reserve the `router-status` subdomain (relative to your platform domain) for use by the router (i.e. add it to the short list of names that the controller doesn't allow for user apps). Then router status could consistently be accesses at `router-status.example.com`...

We'd probably want to enable whitelists on that endpoint as well.